docs/how-to-guides/authentication.md

```diff
@@ -0,0 +1,7 @@
+By default, Discourse will be set up to create users locally, in the PostgreSQL database configured for the application. For local testing it's usually easiest to create a custom admin user via the rails console than to configure an email server that can send you a confirmation email. To do this, simply get a shell on the discourse workload pod, and then run:
+```
+cd /srv/discourse/app && ./bin/bundle exec rake admin:create RAILS_ENV=production
```

Comment:
Would it be worth mentioning that this can be done in automation using `cd /srv/discourse/app && ./bin/bundle exec rake admin:create RAILS_ENV=production << ANSWERS\n$EMAIL\n$PASSWORD\n$PASSWORD\nY\nANSWERS`?

Reply:
I'm removing this section since the [action to create the user](https://github.com/canonical/discourse-k8s-operator/pull/23) will land soon